pipelines/skylab/smartseq2_single_nucleus_multisample/human_single_example.json

```diff
@@ -0,0 +1,18 @@
+{
+    "MultiSampleSmartSeq2.genome_ref_fasta": "gs://gcp-public-data--broad-references/hg38/v0/GRCh38.primary_assembly.genome.fa",
+    "MultiSampleSmartSeq2.rrna_intervals": "gs://gcp-public-data--broad-references/hg38/v0/gencode.v27.primary_assembly.annotation.interval_list",
+    "MultiSampleSmartSeq2.gene_ref_flat": "gs://gcp-public-data--broad-references/hg38/v0/gencode.v27.primary_assembly.annotation.refflat.txt",
+    "MultiSampleSmartSeq2.hisat2_ref_name": "hisat2_primary_gencode_human_v27",
```

Comment:
Generally I don't think this inputs json has been updated for single nucleus - the name of the workflow is different and it references hisat2

Reply:
We do not have any human examples yet. Will update it once we get the data. I removed these files for now.